devfile.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: 1.0.0
+metadata:
+  name: che-plugin-broker
+projects:
+  - name: che-plugin-broker
+    source:
+      location: 'https://github.com/eclipse/che-plugin-broker.git'
+      type: git
+      branch: master
+    clonePath: src/github.com/eclipse/che-plugin-broker
+components:
+  - id: ms-vscode/go/0.11.0
+    type: chePlugin
+    alias: ms-vscode.go
+
+  - type: dockerimage
+    image: 'eclipse/che-plugin-broker-dev:next'
```

Comment:
By default the `.ci/build.sh` script builds images tagged `latest` and `${TAG}`, should this be updated to `latest`?

Reply:
Fixed. Good catch.